web/src/ui_init.js

```diff
@@ -708,6 +707,9 @@ export function initialize_everything() {
     spoilers.initialize();
     lightbox.initialize();
     click_handlers.initialize();
+    // Popovers module should be initialized after the right sidebar module
+    // to correctly register popover events.
+    popovers.initialize();
```

Comment:
So I think the alternative here would be to just change the presence-related click handlers in question to be registered on `body`, like we do for the other 90% of click handlers in `popover.js`. I think that might be a better solution?